tests/models/test_models_logprobs.py

```diff
@@ -0,0 +1,90 @@
+"""Compares the outputs of hf vs vllm for medium sized models.
+
+There is not bitwise correctness for fp16 inference.
+As a result, in this test, we just confirm that the top selected tokens of the
+Marlin/GPTQ models are in the top 3 selections of each other.
+
+Run `pytest tests/models/test_models_medium_logprobs.py`.
+"""
+import pytest
+import torch
+
+from tests.models.utils import check_logprobs_close
+
+MAX_MODEL_LEN = 1024
+
+MODELS = [
```

Comment:
How long does it take to run all models listed here? Can some of them be removed to reduce the CI time?

Reply:
I can just remove them. I just wanted to prove the accuracy was right